blocks/library/embed/index.js

```diff
@@ -26,18 +26,13 @@ import BlockAlignmentToolbar from '../../block-alignment-toolbar';
 // These embeds do not work in sandboxes
 const HOSTS_NO_PREVIEWS = [ 'facebook.com' ];
 
-function getEmbedBlockSettings( { title, icon, category = 'embed', transforms, keywords = [] } ) {
+function getEmbedBlockSettings( { title, description, icon, category = 'embed', transforms, keywords = [] } ) {
 	return {
 		title,
-
-		description: __( 'The Embed block allows you to easily add videos, images, tweets, audio, and other content to your post or page.' ),
-
+		description: description || __( `Paste URLs from ${ title } to embed the content in this block.` ),
```

Comment:
Because of how string extraction works, we cannot do variable string interpolation for translated strings. 

This is the same as it is in PHP:

https://codex.wordpress.org/I18n_for_WordPress_Developers#Placeholders

We should use a placeholder here, and `wp.i18n.sprintf` to do the variable replacement.

Reply:
@aduth my bad missed that part have updated the same here #6354